src/components/panes/CallPane.jsx

```diff
@@ -70,9 +70,12 @@ export default class CallPane extends PaneBase {
         }
     }
 
-    componentWillReceiveProps(nextProps) {
+     componentWillReceiveProps(nextProps) {
```

Comment:
Is this whitespace change necessary, or perhaps a typo?

Reply:
Oops! Fixed by bede541